### PR TITLE
Implement new ship: Sunder

### PR DESCRIPTION
**Pull request by eebop**
**Content**

## Summary
This implements the Sunder as a cheap (ish) human mining ship. The artwork was created in [PR 6895](https://github.com/endless-sky/endless-sky/pull/6895)

## Save File
[test2 test2.txt](https://github.com/endless-sky/endless-sky/files/9012486/test2.test2.txt)

## Testing Done
The ship seems well-balenced as a light freighter with some mining capacity (i. e. strong engines). Although it has some weapons (to break the asteroids, it loses in combat to an intercepter.

## Performance Impact
N/A

### CODE REVIEW

**Comment by Saugia** [comment]
I believe @Amazinite  had mentioned he wants to implement a few new things along with the Sunder, such as mining lasers, some type of tractor turret, etc. when he gets to it. The drones in that PR were also specifically designed for the Sunder, as the model itself has a rack for 2 drones. In the image of that PR you can see it carrying two of them underneath. These too would have the new mining laser. 

I believe this is a little premature addition of the ship, because of all the new implementations that are meant to come with it. Perhaps it can still be implemented like this as a basic representation before we get to the new content for it? Either way, we'll have to see what Derpy says as he specifically asked me to make these models for him.

**Comment by Hecter94** [comment]
> For some reason, every tab in ships.txt got deleted automatically? If this is a problem I can go in and add them all back

What IDE are you using to make changes? Issues with tabbing are often the result of using a word processor rather than an IDE like Sublime, Eclipse or N++

Those tab issues will need to be fixed before merging can be considered as it seems to be causing a lot of merge conflicts and makes reviewing the PR difficult. 
I'd suggest making a copy of just your changes, reverting the file back to vanilla and then using a program like the ones mentioned above to add your changes back in to avoid any extraneous file edits.

**Comment by Amazinite** [comment]
As @Saugia mentioned, there's a lot more needed for this ship than just the stats. It also needs the drone, various outfits that we either don't have sprites and stats for or just outright aren't possible in the game right now, and preferably the various AI changes that Sam has open PRs for.

**Comment by MasterOfGrey** [comment]
> I was already going to implement the mining drone and add a mining laser that I had been thinking about for a while.

Sounds like you might save some people some work though here. :)

**Comment by eebop** [comment]
All that's needed now is some kind of escorts:mine instruction and some graphics (and proof reading). One thing that this adds that I'm not so sure about is the mining laser also doubles as a very very weak shield-piercing laser so i don't know if that'll get aproved.

**Comment by eebop** [comment]
One important pr I'm waiting on is #6840

**Comment by Zitchas** [comment]
Given that this is a human ship, and an early game so at that, I really don't think the shield piercing is appropriate. At best that might be a post automata invasion development.

**Comment by Hurleveur** [comment]
The description of the weapon itself seems too advanced for human tech level, I think.
We don't even have tractor beams in game yet, and even if we did this seems like a t2-t3 tech, manipulating gravity with a null field or whatever doesn't seem easy. Remnants could maybe develop something like too, this with their advanced scientific knowledge.

This weapon does such a negligible amount of damage that piercing or not won't matter anyway. I very much like the idea of it, but I'd imagine this to be the way Quargs/Exiles mine things.

File: data/human/ships.txt [hunk 1]
```diff
@@ -8,6 +8,41 @@
...
+
+		"X1700 Ion Thruster"
+		"X1200 Ion Steering"
+
+	gun 0 0
+	engine -5 29
+	engine 5 29
+	explode "tiny explosion" 15
+	"final explode" "final explosion small"
+	description "Thing."
```
Comment (warp-core) [inline data/human/ships.txt:44]: This is gonna need a little more.

**Review by warp-core — COMMENTED** [review]
There's a lot of whitespace changes that shouldn't be here.


The Mining Drone is better *in every way* than the Combat Drone.
This could perhaps be an argument to buff the Combat Drone, but then I'd say the Flea needs buffing as well.
My suggestions are:
- Remove the shields
- Reduce the hull
- Increase mass
- Reduce the outfit space
- Add some cargo space

**Review by warp-core — CHANGES_REQUESTED** [review]
Sorry, I hit the wrong button before, didn't mean to submit then.
This Repulsion Laser has a number of problems:
- Inappropriate thumbnail
- Called a 'laser' but behaves like a blaster.
- Doesn't have unique projectile sprite, sound, or effect (not necessarily an issue, but doesn't seem great to use Energy Blaster  effects since it'll make it difficult to tell the two apart, and they are likely to appear in similar places).
- Way too big for the drone you've mounted it on.
- Probably shouldn't do piercing.
- As a ship to ship weapon, it's not very performant, and narratively, the technology described in the description probably doesn't have much place in human space at this stage.

Personally, I would just remove this weapon from this PR and use a Beam Laser on the drone (until an actual mining laser using planned features is implemented).

**Comment by Amazinite** [comment]
As I mentioned earlier, there's a lot more that the Sunder needs than just stats. There are a number of other PRs that first need finalized and merged that add new mechanics or AI behaviors that the Sunder will take advantage of, and without those the Sunder won't really work as it's intended to. Aside from that, the proposed stats for the Sunder, mining drone, and mining laser are rather far off from what I believe they should be. So given that, I'd rather just close this and revisit the Sunder once it's actually ready to be implemented.

**Pull request closed without merging**